### PR TITLE
fix: footer  items and validation

### PR DESCRIPTION
**Pull request by abdullai-t**
####  Summary / Highlights
The most recent production release `v6.2.2` introduced a number of bugs. This pull request addresses the following

- [x] Fix the issue with the footer menu not displaying.
- [x] Ensure user-submitted testimonials create or update correctly even when no vendors are present.
- [x] Update community impact data properly, even when certain field values are empty.

#### Requirements (place an `x` in each `[ ]`)
* [x] I've read and understood the [Contributing Guidelines](/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've tested my changes manually.
* [ ] I've added unit tests to my PR.

##### Transparency ([Project board](https://github.com/orgs/massenergize/projects/7/views/2))
* [x] I've given my PR a meaningful title.
* [x] I linked this PR to the relevant issue.
* [ ] I moved the linked issue from the "Sprint backlog" to "In progress" when I started this.
* [ ] I moved the linked issue to "QA Verification" now that it is ready to merge.

### CODE REVIEW

**Review by BradHN1 — APPROVED** [review]
LGTM

**Review by Opoku-Agyemang — APPROVED** [review]

File: src/_main_/utils/common.py [hunk 1]
```diff
@@ -153,7 +153,7 @@ def parse_string(s):
 
 def parse_int(b):
     if not str(b).isdigit():
-        raise ValueError("Input must be a digit")
+        return None
```
Comment (Opoku-Agyemang) [inline src/_main_/utils/common.py:156]: This is strange.  If we return None then do we check if its None in the validator?

Comment (abdullai-t) [reply src/_main_/utils/common.py:156]: no

Comment (abdullai-t) [reply src/_main_/utils/common.py:156]: I have added that now

**Pull request merged**